data/xml/2010.amta.xml

```diff
@@ -0,0 +1,694 @@
+    </paper>
+  </volume>
+  <volume id="tutorials" ingest-date="2021-05-22">
+    <meta>
+      <booktitle>Proceedings of the 9th Conference of the Association for Machine Translation in the Americas: Tutorials</booktitle>
+      <publisher>Association for Machine Translation in the Americas</publisher>
+      <address>Denver, Colorado, USA</address>
+      <month>October 31-November 4</month>
+      <year>2010</year>
+    </meta>
+    <paper id="1">
+      <title>Introduction to Post-Editing: Who, What, How and Where to Next?</title>
+      <author><first>Sharon</first><last>O’Brien</last></author>
+      <url hash="f6d194d4">2010.amta-tutorials.1</url>
+    </paper>
+    <paper id="2">
+      <title>An Introduction to Machine Translation</title>
+      <author><first>Mike</first><last>Dillinger</last></author>
+      <url hash="0bb1f86f">2010.amta-tutorials.2</url>
+    </paper>
+    <paper id="3">
+      <title><fixed-case>MT</fixed-case> Postediting Showcase</title>
+      <author><first>Laurie</first><last>Gerber</last></author>
+      <author><first>Elliott</first><last>Macklovitch</last></author>
+      <url hash="1dda3388">2010.amta-tutorials.3</url>
+    </paper>
+    <paper id="4">
+      <title>Evaluating the Output of Machine Translation Systems</title>
+      <author><first>Alon</first><last>Lavie</last></author>
+      <url hash="ddfdfacb">2010.amta-tutorials.4</url>
+    </paper>
+    <paper id="5">
+      <title>Machine Translation with Open source Software</title>
+      <author><first>Philipp</first><last>Koehn</last></author>
+      <author><first>Hieu</first><last>Hoang</last></author>
+      <url hash="531c99d0">2010.amta-tutorials.5</url>
+    </paper>
+  </volume>
+  <volume id="technology" ingest-date="2021-05-22">
+    <meta>
+      <booktitle>Proceedings of the 9th Conference of the Association for Machine Translation in the Americas: Technology Showcase</booktitle>
+      <publisher>Association for Machine Translation in the Americas</publisher>
+      <address>Denver, Colorado, USA</address>
+      <month>October 31-November 4</month>
+      <year>2010</year>
+    </meta>
+    <paper id="1">
+      <title>Exhibit Descriptions</title>
+      <author><first>Jennifer</first><last>DeCamp</last></author>
+      <url hash="b6d7442c">2010.amta-technology.1</url>
+    </paper>
```

Comment:
We could also just process this as a complete volume (`<url>` tag in the `<meta>` block). Not sure which is best.